lib/dal/src/job/definition/compute_validation.rs

```diff
@@ -107,23 +109,34 @@ impl JobConsumer for ComputeValidation {
                 continue;
             }
 
-            let value = AttributeValue::get_by_id(ctx, av_id)
-                .await?
-                .value(ctx)
+            let ctx_clone = ctx.clone();
```

Comment:
Most places I've seen we just use the same variable name (makes the code read the same whether it's in the block or not.)
```suggestion
            let ctx = ctx.clone();
```